### PR TITLE
TD-3265 Incorrect font in Plotly plots

### PR DESCRIPTION
**Pull request by mattcorner**
Contributes to [TD-3265](https://sce.myjetbrains.com/youtrack/issue/TD-3265/Plotly-incorrect-font)

## Changes

@Sowbhagya-ipg noted that the Plotly components are not always rendering the correct fonts. After some digging I noted that we are applying fonts to specific elements of the component rather the whole lot, and sometimes Plotly appears to be overriding those to be `"Open Sans", verdana, arial, sans-serif`.

The fonts are now applied to the component as a whole and this seems to work well.

## UI/UX

`<BulletGauge />`
Before: https://main--666c1309face6e7c27f20e04.chromatic.com/?path=/story/plots-bulletgauge--default
Now: https://666c1309face6e7c27f20e04-hdvpoqhluf.chromatic.com/?path=/story/plots-bulletgauge--default

`<LinePlot />`
Before: https://main--666c1309face6e7c27f20e04.chromatic.com/?path=/story/plots-lineplot--default
Now: https://666c1309face6e7c27f20e04-hdvpoqhluf.chromatic.com/?path=/story/plots-lineplot--default

`SurfacePlot />`
Before: https://main--666c1309face6e7c27f20e04.chromatic.com/?path=/story/plots-surfaceplot--default
Now: https://666c1309face6e7c27f20e04-hdvpoqhluf.chromatic.com/?path=/story/plots-surfaceplot--default

## Testing notes

Check that the font is consistent across the plots and is Montserrat.

@Sowbhagya-ipg Can you check these as the person most affected by this? I think your fallback font is quite different to Montserrat.

## Author checklist

Before I request a review:

- [x] I have reviewed my own code-diff.
- [x] I have tested the changes in Docker / a deploy-preview.
- [x] I have assigned the PR to myself or an appropriate delegate.
- [x] I have added the relevant labels to the PR.
- ~I have included appropriate tests.~
- [x] I have checked that the Lint and Test workflows pass on Github.
- ~I have populated the deploy-preview with relevant test data.~
- [x] I have tagged a UI/UX designer for review if this PR includes UI/UX changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -104,14 +105,10 @@
     "@mui/icons-material": "^5.16.4",
     "@mui/x-data-grid": "^7.10.0",
     "@mui/x-tree-view": "^7.10.0",
-    "@types/css-mediaquery": "^0.1.4",
-    "@types/plotly.js": "^2.33.0",
-    "@types/react-plotly.js": "^2.6.3",
-    "@types/react-text-mask": "^5.4.14",
```
Comment (mattcorner) [inline package.json:110]: These devDependencies were incorrectly duplicated in dependencies.

**Comment by Sowbhagya-ipg** [comment]
Looks good matt. Checked in the storybook.

File: src/BulletGauge/BulletGauge.tsx [hunk 2]
```diff
@@ -50,6 +50,7 @@ const BulletGauge = ({ title, value, suffix }: BulletGaugeProps) => {
       ]}
       layout={{
         font: {
+          family: "Montserrat, sans-serif",
```
Comment (dmbarry86) [inline src/BulletGauge/BulletGauge.tsx:53]: Any reason this component includes the sans-serif family fallback but the other two don't?

Comment (dmbarry86) [reply src/BulletGauge/BulletGauge.tsx:53]: All looks fine apart from this question.

**Pull request merged**